CHANGELOG.md

```diff
@@ -7,15 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [v1.0.0] - 2020-02-07
+
 ## [v1.1.0] - 2020-04-22
 ### Added
 - New module `splitout`
 ### Updated
 - Use Python 3 for local action modules
 - Various fixes
+
 ## [v1.2.0] - 2020-05-19
 - Removed resize functionality from redologs module and moved it to utils GUI
 
+## [v1.3.0] - 2020-05-19
```

Comment:
This can all be tracked under the v1.2.0 release since it's part of the same PR.